cluster/addons/dns/coredns/coredns.yaml.base

```diff
@@ -64,9 +64,9 @@ data:
     .:53 {
         errors
         health
+        ready
```

Comment:
Probably better to split the CoreDNS bump into a separate commit?

Reply:
Done